OpenRA.Game/Map/Map.cs

```diff
@@ -217,12 +217,24 @@ static object LoadVideos(MiniYaml y)
 
 		[FieldLoader.Ignore] public Lazy<Dictionary<string, ActorReference>> Actors;
 
-		public int PlayerCount { get { return Players.Count(p => p.Value.Playable); } }
+		public int PlayerCount { get { return GetPlayers().Count(p => p.Value.Playable); } }
+
+		public Dictionary<string, PlayerReference> GetPlayers()
```

Comment:
It would be nice if you could decouple this even further from Map.  Maybe create a separate MapPlayers class that you can initialize with a map and has a .Save() method.